apps/scan/backend/src/scanners/custom/state_machine.ts

```diff
@@ -370,6 +371,11 @@ function storeInterpretedSheet(
   return addedSheetId;
 }
 
+/**
+ * A mutex to ensure that continuous export operations happen sequentially and do not interleave
+ */
+const continuousExportMutex = new Mutex(0);
```

Comment:
Created https://github.com/votingworks/vxsuite/pull/4368 so you don't have to provide a dummy value.